hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/TestSerializerHook.java

```diff
@@ -18,34 +18,32 @@
 
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
-import com.hazelcast.nio.serialization.Serializer;
-import com.hazelcast.nio.serialization.SerializerHook;
 import com.hazelcast.nio.serialization.StreamSerializer;
 
 import java.io.IOException;
 
 /**
  * Test example of a serializer hook
  */
-public class TestSerializerHook implements SerializerHook {
+public class TestSerializerHook  {
 
     public TestSerializerHook() {
     }
 
-    @Override
-    public Class getSerializationType() {
-        return SampleIdentifiedDataSerializable.class;
-    }
-
-    @Override
-    public Serializer createSerializer() {
-        return new TestSerializer();
-    }
-
-    @Override
-    public boolean isOverwritable() {
-        return true;
-    }
+//    @Override
+//    public Class getSerializationType() {
+//        return SampleIdentifiedDataSerializable.class;
+//    }
+//
+//    @Override
+//    public Serializer createSerializer() {
+//        return new TestSerializer();
+//    }
+//
+//    @Override
+//    public boolean isOverwritable() {
+//        return true;
+//    }
```

Comment:
Leftover?

Reply:
Bingo :) Seems when running 3.x and 4.x in the same JVM, I get into some weird classloader issues with this hook, and it was irrelevant for WAN testing, which is why I commented it out when running the tests.

Reply:
Fixed.